rust/src/config.rs

```diff
@@ -38,17 +40,31 @@ const fn default_naptime() -> f64 {
 }
 
 /// Auth entries, which are composed of a [`User`](nix::unistd::User) and its associated [`Group`](nix::unistd::Group).
-/// The user and the group have the same name and the user should be part of the associated group.
-///
+/// The user should be part of the associated group.
 #[derive(Debug, Deserialize)]
 pub struct DummyAuthConfig {
-    pub entries: [String; 2],
+    pub entries: [(String, String); 3],
 }
 
 impl Default for DummyAuthConfig {
     fn default() -> Self {
         Self {
-            entries: [String::from("nobody"), String::from("tests")],
+            entries: [
+                (
+                    String::from("nobody"),
+                    if let Ok(Some(nobody)) = User::from_name("nobody") {
+                        if let Ok(Some(group)) = Group::from_gid(nobody.gid) {
+                            group.name
+                        } else {
+                            String::from("nobody")
+                        }
+                    } else {
+                        String::from("nobody")
+                    },
+                ),
```

Comment:
While this would work, I'm not sure it's worth handling the case where `User::from_name("nobody")` fails.  Simpler just to do
```suggestion
                    Group.from_gid(User::from_name("nobody").unwrap().gid).unwrap()
```